lib/src/over_react_redux/devtools/middleware.dart

```diff
@@ -36,23 +36,25 @@ class _ReduxDevToolsExtensionConnection {
 @JS('__REDUX_DEVTOOLS_EXTENSION__.connect')
 external _ReduxDevToolsExtensionConnection reduxExtConnect([dynamic options]);
 
+final Logger log = Logger('OverReactReduxDevToolsMiddleware')
+  ..onRecord.listen((rec) {
```

Comment:
This listener will now get set up lazily when `log` is initialized, the first time `log` is accessed

Reply:
Can do 👍 

```suggestion
  // This listener gets set up when `log` is lazy-initialized, the first time it's accessed.
  ..onRecord.listen((rec) {
```